### DISCUSSION TITLE
yubikey/fido2

### DISCUSSION THREAD

**Discussion by TheAkashicTraveller** [bromite/bromite#1040] [General]
Is there any particular reason I can use my yubikey in chrome but not in bromite? If we can get proper support with the webview replacement we'd actually have pretty much full support on the android side.

**Comment by TheAkashicTraveller** [comment]
What I'm hoping for is webauthn support.

**Comment by csagan5** [comment]
> Is there any particular reason I can use my yubikey in chrome but not in bromite?

Anything which requires using Play Store integrations is disabled in Bromite because it would require compiling with binary blobs and the data collected and sent to Google servers would not be known a-priori.

**Reply by TheAkashicTraveller** [reply]
So google made that part closed source? Great /s
